TaskService/Task.cs

```diff
@@ -1074,12 +1074,13 @@ public RunningTaskCollection GetInstances()
 			throw new NotV1SupportedException();
 		}
 
-		/// <summary>
-		/// Gets the last registration time, looking first at the <see cref="TaskRegistrationInfo.Date"/> value and
-		/// then looking for the most recent registration event in the Event Log.
-		/// </summary>
-		/// <returns><see cref="DateTime"/> of the last registration or <see cref="DateTime.MinValue"/> if no value can be found.</returns>
-		public DateTime GetLastRegistrationTime()
+#if !NETSTANDARD

```

Comment:
For this, let's just put the conditions around the
`if (ret == DateTime.MinValue)` section. This property will then still be valid, it just won't go to the Event Log for more detail.